src/GraphQl/Serializer/ItemNormalizer.php

```diff
@@ -62,11 +65,24 @@ public function supportsNormalization($data, $format = null)
         return self::FORMAT === $format && parent::supportsNormalization($data, $format);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function denormalize($data, $class, $format = null, array $context = [])
+    {
+        return $this->itemNormalizer->denormalize($data, $class, $format, $context);
```

Comment:
Can't you use `parent::` directly? It feels weird to me to inject an item normalizer in another one.

Reply:
I can't because the default item normalizer is final. And I need it because I don't want to duplicate the code needed to update related existing resources.

Reply:
Remove the `final` keyword and add a `@final` PHPDoc annotation then. It will be cleaner IMHO.